core-web/libs/sdk/angular/src/lib/layout/container/container.component.ts

```diff
@@ -1,65 +1,65 @@
 import {
-  ChangeDetectionStrategy,
-  Component,
-  Input,
-  OnChanges,
-  computed,
-  inject,
-  signal,
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnChanges,
+    computed,
+    inject,
+    signal
 } from '@angular/core';
-import { AsyncPipe, NgComponentOutlet } from '@angular/common';
 
-import { getContainersData } from '../../utils';
-import {
-  PageContextService,
-} from '../../services/dotcms-context/page-context.service';
 import { NoComponentComponent } from '../../components/no-component/no-component.component';
 import { DotCMSContainer, DotCMSContentlet, DynamicComponentEntity } from '../../models';
+import { PageContextService } from '../../services/dotcms-context/page-context.service';
+import { getContainersData } from '../../utils';
 
 interface DotContainer {
-  acceptTypes: string;
-  identifier: string;
-  maxContentlets: number;
-  uuid: string;
-  variantId?: string;
+    acceptTypes: string;
+    identifier: string;
+    maxContentlets: number;
+    uuid: string;
+    variantId?: string;
 }
 
 const EMPTY_CONTAINER_EDIT_MODE_STYLES = {
-  width: '100%',
-  backgroundColor: '#ECF0FD',
-  display: 'flex',
-  justifyContent: 'center',
-  alignItems: 'center',
-  color: '#030E32',
-  height: '10rem',
+    width: '100%',
+    backgroundColor: '#ECF0FD',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#030E32',
+    height: '10rem'
 };
 
 @Component({
-  selector: 'dotcms-container',
-  standalone: true,
-  imports: [AsyncPipe, NgComponentOutlet, NoComponentComponent],
-  templateUrl: './container.component.html',
-  styleUrl: './container.component.css',
-  changeDetection: ChangeDetectionStrategy.OnPush,
+    selector: 'dotcms-container',
+    standalone: true,
+    imports: [AsyncPipe, NgComponentOutlet, NoComponentComponent],
+    templateUrl: './container.component.html',
+    styleUrl: './container.component.css',
+    changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class ContainerComponent implements OnChanges {
     @Input({ required: true }) container!: DotCMSContainer;
 
     private readonly pageContextService: PageContextService = inject(PageContextService);
-    protected readonly emptyContainerStyles: Record<string, string> = EMPTY_CONTAINER_EDIT_MODE_STYLES;
+    protected readonly emptyContainerStyles: Record<string, string> =
+        EMPTY_CONTAINER_EDIT_MODE_STYLES;
     protected readonly NoComponentComponent = NoComponentComponent;
     protected readonly isInsideEditor = signal<boolean>(false);
 
     protected componentsMap!: Record<string, DynamicComponentEntity>;
     protected contentlets = signal<DotCMSContentlet[]>([]);
     protected dotContainer = signal<DotContainer | null>(null);
-    protected dotContainerAsString = computed(() =>
-        JSON.stringify(this.dotContainer())
-    );
+    protected dotContainerAsString = computed(() => JSON.stringify(this.dotContainer()));
```

Comment:
Why do we do this?

Reply:
The client needs the container data stringified in the contentlet